### ISSUE TITLE
Dopamine becomes slower with every new song played

### ISSUE THREAD

**Issue by Parzival5129** [digimezzo/dopamine#386]
Everytime I start playing different songs in a short time (manually), Dopamine needs more time to start playing them. A simple restart of the program fixes the issue until I played a few songs again. Then, it needs several seconds to play a new title again

**Comment by iPodVideo** [comment]
How large are the files you're playing?

**Comment by Parzival5129** [comment]
> How large are the files you're playing and what type of storage are you playing them off of? (Spinning HDD, SSD, Flash Drive)

~20mb, playing from an SSD. If the speed of the drive would be the issue, the songs would always take a while to load and not after some time

**Comment by iPodVideo** [comment]
Yeah I don't think this is an issue with your storage or the size of the files. I gave it a go on my SSD using small MP3s and it exhibited the same behaviour as my HDD with big flacs

**Comment by Rushikesh-Malave-175** [comment]
turn off the visualliser. Somtimes that causes the app to slow down at the beginning
